src/index.js

```diff
@@ -1 +1,63 @@
 import './style.css';
+
+const scoreList = document.getElementById('scorelist');
+const userName = document.getElementById('user');
+const scoreMark = document.getElementById('score');
+const submitButton = document.getElementById('submitButton');
+const refreshButton = document.getElementById('refreshButton');
+
+let scores = [];
+const apikey = 'AKAJVPw0U06VBfFrk32s';
+const displayList = async () => {
+  const response = await fetch(
+    `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${apikey}/scores/`,
+  );
+  scores = await response.json();
+  if (scores.result.length) {
+    scores.result.forEach((val) => {
+      const li = document.createElement('li');
+      li.classList.add('list-group-item');
+      const text = `
+      <div class="d-flex justify-content-between align-items-start" id="${val.index}">
+        <span class="fw-bold">
+          ${val.user}
+        </span>
+        <span class="fw-bold">
+        ${val.score}
+        </span>
+      </div>`;
+      li.innerHTML = text;
+      scoreList.appendChild(li);
+    });
+  }
+};
+
```

Comment:
It would be better to remove the hard-coded elements and append the new elements of the leaderboard dynamically. That way, it would be much easier to maintain in the future.